climate_scanner/coarse_article_classifier/predict.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import os
+import torch
+import torch.nn as nn
+from transformers import BertTokenizer
+from transformers import BertModel
+
+_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+def load_weights(model, path):
+    path1 = os.path.join(_ROOT, path)
+    model.load_state_dict(torch.load(path1))
+    return model
+
+def convert_probs_to_labels(probs, threshold = 0.5):
+    labels = np.where(probs > threshold, 1, 0)
+    return labels
+
+class BertClassifierModel(nn.Module):
```

Comment:
Again, is this not functionally identifcal to the version in the coarse_class.py, why not just import the class from the other file?